src/store/facilityList/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface FacilityList {
+  capacity: undefined;
+  currentPopulation: undefined;
+}
```

Comment:
should be type `number`

Reply:
had to also change reducer.ts objects to 0 for the type to be number in type.ts and TotalUsers.tsx

Heres the dashboard with the data from: /facility/by/602ea8d423a00b4812b77ee6

<img width="1791" alt="Screen Shot 2021-02-21 at 2 08 32 AM" src="https://user-images.githubusercontent.com/43226793/108618175-e6b30c00-73e9-11eb-9aac-acd5d20d9577.png">

Reply:
will also fix styling in another pr